index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <img src="resources/images/Logo.png" alt="my logo">
+        <nav class="menu">
+        <img src="resources/icons/menu.png" alt="menu icon">
+    </nav>
+    </header>
+    <section class="headline-container">
+        <article class="article">
+          <div class="article item1">
+            <h1 class="title">
+              I’m Afsheen Kiran. <br />
+              Glad to see you!
+            </h1>
+          </div>
+          <div class="article item2">
+            <p class="presentation">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
```

Comment:
Please kindly use the right font color for your paragraph similar to what we have in the design which is `#344563`